### ISSUE TITLE
the defaut setting for threads is not safe

### ISSUE THREAD

**Issue by jvollme** [soedinglab/MMseqs2#539]
The default setting for threads of mmseqs2 is to just use all threads. This is not a safe default, especially if multiple people are working on the same system. One student who does not read the manual CAREFULLY can crash everything for everyone.
Would you consider setting the default to 1?
I realize this slows things down for people who DON'T read the manual/help-funciton. But i believe it is better to have safe defaults and mildly punish the people who DON'T read instructions than to have convenient defaults that severely punish everybody else...

**Comment by zhou-sumei** [comment]
I agree with you. especially for the programe in the high performance computer

**Comment by martin-steinegger** [comment]
Closing this one due to inactivity — it was answered above and we haven't heard back. If it's still relevant, please reopen with your MMseqs2 version, the exact command, and the full output, and we'll be happy to help. Thanks!
